runtime/src/accounts_index.rs

```diff
@@ -219,6 +219,16 @@ impl AccountSecondaryIndexes {
             None => true, // include all keys
         }
     }
+    pub fn default_for_test() -> Self {
+        Self {
+            keys: None,
+            indexes: HashSet::from([
+                AccountIndex::ProgramId,
+                AccountIndex::SplTokenMint,
+                AccountIndex::SplTokenOwner,
+            ]),
+        }
+    }
```

Comment:
Is this used anywhere?

Reply:
I don't think anymore. Do you not think it will be useful to have?

Reply:
Now used

Reply:
it's in "test_secondary_index_key_sizes()" but removing.